src/stream/manager.rs

```diff
@@ -36,6 +36,9 @@ pub fn start() {
             StreamType::UDP(stream) => {
                 stream.start();
```

Comment:
I would recommend to change the commit message to only: `src: stream: Add RTSP video stream`
Since you are doing changes in types, manager and backend